### DISCUSSION TITLE
Server Count Firestore

### DISCUSSION THREAD

**Discussion by okaforcj** [firebase/flutterfire#9999] [Q&A] [answered]
Hello, I noticed firebase updated capabilities to allow for getting native count of a collection recently (~1-2months ago), I see the functionality in all the major sdk but there isnt any one for flutter. does anyone know if flutter will be supported anytime soon, or what the reasoning is, behind not supporting it to start with? Any advice on this would be much appreciated

**Comment by Ehesp** [comment, marked as answer]
Count had been supported since it was released: https://pub.dev/documentation/cloud_firestore/latest/cloud_firestore/Query/count.html

**Reply by okaforcj** [reply]
ahh, its not listed in the google webpage https://firebase.google.com/docs/firestore/query-data/aggregation-queries and i hadnt updated my pubspec, thanks for point it out
